cmake/alpakaCommon.cmake

```diff
@@ -888,5 +888,10 @@ if((ALPAKA_ACC_GPU_CUDA_ENABLE OR ALPAKA_ACC_GPU_HIP_ENABLE) AND ALPAKA_CUDA_COM
                  PROPERTY INTERFACE_COMPILE_OPTIONS)
     string(REPLACE ";" " " _ALPAKA_COMPILE_OPTIONS_STRING "${_ALPAKA_COMPILE_OPTIONS_PUBLIC}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_ALPAKA_COMPILE_OPTIONS_STRING}")
+
+    # CMAKE_CXX_FLAGS_[Release|Debug|RELWITHDEBINFO] to CMAKE_CXX_FLAGS that nvcc is translating 
```

Comment:
It should be `RelWithDebInfo`, not `RELWITHDEBINFO`. I also propose a rewording of the comment:

```cmake
# Append CMAKE_CXX_FLAGS_[Release|Debug|RelWithDebInfo] to CMAKE_CXX_FLAGS
# because FindCUDA only propagates the latter to nvcc.
```